HISTORY.md

```diff
@@ -1,6 +1,142 @@
 ---
 title: Release History
 ---
+# 3.0.0
+## Highlights
+Mesa 3.0 brings major improvements to agent-based modeling, making it more intuitive and powerful while reducing complexity. This release modernizes core functionalities and introduces new capabilities for both beginners and advanced users.
+
+### Streamlined agent management
+
+The centerpiece of Mesa 3.0 is its new agent management system. Agents are now automatically tracked and assigned unique IDs, eliminating common boilerplate code. The new AgentSet functionality provides an elegant and flexible way to work with agents:
+
+```python
+# Find agents meeting specific criteria
+wealthy_agents = model.agents.select(lambda a: a.wealth > 1000)
+
+# Group and analyze agents
+grouped = model.agents.groupby("species")
```

Comment:
```suggestion
grouped = model.agents.groupby(type)
```

the example smells to much like netlogo style breed, so if you want to give a class based example, using the builtin type is better. If you want to give an attribute based example, you might use someting like 

```suggestion
state_stats = model.agents.groupby("state").agg({
     "count": len,
     "avg_age": ("age", np.mean),
     "total_wealth": ("wealth", sum)
 })
```
of course, because of chaining you can also do this as a one-liner:

```suggestion
grouped = model.agents.groupby("state")
state_stats = grouped.agg({
     "count": len,
     "avg_age": ("age", np.mean),
     "total_wealth": ("wealth", sum)
 })
```

Reply:
Thanks, let's go for `"state"` here split in two lines, I think that's the clearest example.